Mathlib/Geometry/Manifold/ContMDiff/Defs.lean

```diff
@@ -128,12 +128,19 @@ theorem contDiffWithinAt_localInvariantProp (n : ℕ∞) :
     have A : (I' ∘ f ∘ I.symm) (I x) ∈ I'.symm ⁻¹' e'.source ∩ range I' := by
       simp only [hx, mfld_simps]
     have := (mem_groupoid_of_pregroupoid.2 he').1.contDiffWithinAt A
-    convert (this.of_le (mod_cast le_top)).comp _ h _
+    convert (this.of_le hmn).comp _ h _
     · ext y; simp only [mfld_simps]
     · intro y hy; simp only [mfld_simps] at hy; simpa only [hy, mfld_simps] using hs hy.1
 
+/-- Being `Cⁿ` in the model space is a local property, invariant under smooth maps. Therefore,
```

Comment:
This should read "under C^n maps", right? Please replace

Reply:
(I would also say "it lifts nicely" instead of "it will lift", but that is minor, perhaps subjective, and pre-existing.)